contrib/seeds/makeseeds.py

```diff
@@ -25,16 +26,15 @@
 
 PATTERN_IPV4 = re.compile(r"^((\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})):(\d+)$")
 PATTERN_IPV6 = re.compile(r"^\[([0-9a-z:]+)\]:(\d+)$")
-PATTERN_ONION = re.compile(r"^([abcdefghijklmnopqrstuvwxyz234567]{16}\.onion):(\d+)$")
+PATTERN_ONION = re.compile(r"^([a-z2-7]{56}\.onion):(\d+)$")
```

Comment:
nit: done in 24417 which we don't have yet. We should probably backport some helpful `makeseeds.py` changes instead of re-implementing them.

Reply:
I think NACK to this request; I added another commit that pulls in the print exception logic; but our make seeds is so diverged, it doesn't make sense to "backport" it. Two changes that were made there have been pulled in